README.md

```diff
@@ -101,6 +101,10 @@ The `scripts` folder in this repository contains scripts that interact with the
 
   Checks all free-text fields in G-Cloud services for "bad words" and generates a CSV report of any bad words found.
 
+* `send_dos_opportunities_email.py`
+
+  **Runs Mon-Fri on Jenkins**. Send emails with new opportunities on a lot to suppliers who are on this lot.
```

Comment:
Might be nice to include the time it runs (I think it's something like 8am or 9am?).